packages/meditrak-server/src/routes/importEntities/getOrCreateParentEntity.js

```diff
@@ -112,10 +112,8 @@ export async function getOrCreateParentEntity(transactingModels, entityObject, c
       parentEntity,
       transactingModels,
     );
-    if (!parentGeographicalArea)
-      throw new Error(
-        `Parent entity must have geographical area, parent entity code: ${parentCode}`,
-      );
+    if (!parentGeographicalArea) return { parentEntity };
```

Comment:
Should still throw the error if entity is of type facility

Reply:
If you look at the calling code, this is the only place that parent geographical area is used
```
const { parentGeographicalArea, parentEntity } = await getOrCreateParentEntity(
      transactingModels,
      entityObject,
      country,
    );
    if (entityType === transactingModels.entity.types.FACILITY) {
      const defaultTypeDetails = getDefaultTypeDetails(facilityType);
      const facilityToUpsert = {
        type: facilityType,
        type_name: typeName || undefined, // Ensure empty string is treated as undefined
        category_code: categoryCode || undefined, // Ensure empty string is treated as undefined
        code,
        name,
        geographical_area_id: parentGeographicalArea.id,
      };
```

The validation could also be done there instead.